packages/@okta/vuepress-site/docs/guides/device-context/main/nodeexpress/integrationsteps.md

```diff
@@ -0,0 +1,32 @@
+### 1. Capture device ID from your users
+
+Capture the device ID from the user sign-in. When you use the Embedded SDK, it is your responsibility to generate and store these device ID values. These IDs are often UUIDs that take the following format: `123e4567-e89b-12d3-a456-426655440000` and are unique per device. These IDs must be unique per user's device, or unpredictable behavior can result.
```

Comment:
But where do you find these UUIDs? When you say "capture", what does that mean? Do I need to use F12 browser tools, charles, something else?